x-pack/plugins/profiling/public/components/topn_functions.tsx

```diff
@@ -228,6 +241,10 @@ export const TopNFunctionsTable = ({
     <>
       <EuiText size="xs">
         <strong>{totalSampleCountLabel}:</strong> {totalCount}
+        {TotalDiff({
+          samples1: comparisonTopNFunctions?.TotalCount ?? 0,
+          samples2: topNFunctions?.TotalCount ?? 0,
```

Comment:
Is `topNFunctions?.TotalCount ?? 0` the same value as `totalCount`? If so, can we replace it?

Reply:
God spot, fixed.